.run/Run ORT Server.run.xml

```diff
@@ -2,9 +2,11 @@
   <configuration default="false" name="Run ORT Server" type="JetRunConfigurationType">
     <envs>
       <env name="DB_NAME" value="ort_server" />
+      <env name="DB_HOST" value="localhost" />
```

Comment:
Please say in the commit message how you found out that "DB_SSL_MODE" is the correct name to use.

Hint: Probably say that this needs to be align with `docker-compose.yml` / `application.conf` files.

Reply:
You could just say: "This is a fixup for a546302."